sympy/geometry/tests/test_polygon.py

```diff
@@ -405,3 +405,40 @@ def test_intersection():
     assert poly2.intersection(Triangle(Point(0, 1), Point(1, 0), Point(-1, 1))) == [Point(-5/7, 6/7),
                                                                                     Segment(Point2D(0, 1), Point(1, 0))]
     assert poly1.intersection(RegularPolygon((-12, -15), 3, 3)) == []
+
+def test_second_moment_of_area():
+    x, y = symbols('x, y')
+    #triangle
+    p1, p2, p3 = [(0, 0), (4, 0), (0, 2)]
+    p=(0, 0)
+    #equation of hypotenuse
+    eq_y = (1-x/4)*2
+    I_yy = integrate((x**2) * (integrate(1, (y, 0, eq_y))), (x, 0, 4))
+    I_xx = integrate(1 * (integrate(y**2, (y, 0, eq_y))), (x, 0, 4))
+    I_xy = integrate(x * (integrate(y, (y, 0, eq_y))), (x, 0, 4))
+
+    triangle = Polygon(p1, p2, p3)
+
+    i_xx = int(I_xx - triangle.second_moment_of_area(p)[0])
+    i_yy = int(I_yy - triangle.second_moment_of_area(p)[1])
+    i_xy = int(I_xy - triangle.second_moment_of_area(p)[2])
+
+    assert i_xx is 0
+    assert i_yy is 0
+    assert i_xy is 0
```

Comment:
These should be:

```
assert (I_xx - triangle.second_moment_of_area(p)[0]) == 0
```

Reply:
don't use `is` expect if you are checking for the exact object.

Reply:
Ok